vault/logical_system_activation_flags.go

```diff
@@ -34,6 +34,9 @@ This path responds to the following HTTP methods.
 
 	PUT|POST /<feature-name>/activate
 		Activates the specified feature. Cannot be undone.`
+
+	activationFlagIdentityCleanup = "identity-cleanup"
```

Comment:
The name in the RFC and documentation so far was `force-identity-deduplication` which we discussed a bit and preferred because it was a bit more descriptive and because there are other identity things which will not necessarily be "cleaned up" by this.

Reply:
@biazmoreira I saw you pushed a change for the other comment but I think this is the blocking part - we should have this flag named as in the RFC (`force-identity-deduplication`) or else agree on a different name so it is consistent with the docs etc.